tests/configs/test_zoo.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from simsopt.configs import get_QUASR_data
+
+class ZooTests(unittest.TestCase):
+    def test_QUASR_downloader(self):
+        curves, currents, ma = get_QUASR_data(952)
+        coils, ma, surfaces = get_QUASR_data(952, return_style='json')
```

Comment:
Might also be useful to have assertions verifying (from the returned data) that these are the same device record.